studio/components/interfaces/Reports/Reports.constants.ts

```diff
@@ -183,7 +183,44 @@ export const PRESET_CONFIG: Record<Presets, PresetConfig> = {
           avg desc
         limit
         3
-        `
+        `,
+      },
+      networkTraffic: {
+        queryType: 'logs',
+        sql: (filters) => `
+        select
+          cast(timestamp_trunc(t.timestamp, hour) as datetime) as timestamp,
+          coalesce(
+            safe_divide(
+              sum(
+                cast(coalesce(headers.content_length, "0") as int64)
+              ),
+              1000000
+            ),
+            0
+          ) as ingress_mb,
+          coalesce(
+            safe_divide(
+              sum(
+                cast(coalesce(resp_headers.content_length, "0") as int64)
+              ),
+              1000000
+            ),
+            0
+          ) as egress_mb,
+        FROM
+          edge_logs t
+          cross join unnest(metadata) as m
+          cross join unnest(m.response) as response
+          cross join unnest(m.request) as request
+          cross join unnest(request.headers) as headers
+          cross join unnest(response.headers) as resp_headers
+          ${generateRexepWhere(filters)}
```

Comment:
Regex?

Reply:
current report filter operatios (is | match) are built on regexp conditions.

Reply:
ah i didn't even realise the typo 🤦  maybe something wrong with my eyes at night 👀  thanks for the catch haha